DirectEmployers/MyJobs
# [PD 2931] Fixing the hidden apply button in mobile view

Task was to fix the apply button in mobile view for the full calendar when a custom date range is being selected

To Test:

Pull down repository and shrink to mobile view pulling out the full calendar to select a custom date range. The apply button should be visible and not hidden from the primary dimension menu now.
